tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3759,6 +3759,37 @@ public function getLastPublicShareToken():string {
 		}
 	}
 
+	/**
+	 * Returns the attribute values from the last public link share data
+	 *
+	 * @param $attr - attribute name to get
+	 *
+	 * @return string
+	 * @throws Exception
+	 */
+	public function getLastPublicShareAttribute($attr): string {
```

Comment:
```suggestion
	public function getLastPublicShareAttribute(string $attr): string {
```

These days we declare types everywhere that we can.